pkg/controller/bootstrap/common_test.go

```diff
@@ -34,6 +34,7 @@ func newTokenSecret(tokenID, tokenSecret string) *v1.Secret {
 			Namespace:       metav1.NamespaceSystem,
 			Name:            bootstrapapi.BootstrapTokenSecretPrefix + tokenID,
 			ResourceVersion: "1",
+			UID:             "s1",
```

Comment:
make the uid unique by incorporating tokenID (`"uid"+tokenID` or something similar)